mldsa/sign.c

```diff
@@ -685,6 +730,14 @@ int crypto_sign_verify_internal(const uint8_t *sig, size_t siglen,
   /* Call random oracle and verify challenge */
   mld_H(c2, MLDSA_CTILDEBYTES, mu, MLDSA_CRHBYTES, buf,
         MLDSA_K * MLDSA_POLYW1_PACKEDBYTES, NULL, 0);
+
+  /* Constant time: All data in verification is usually considered public.
+   * However, in our constant-time tests we do not declassify the message and
+   * context string.
+   * The following conditional is the only place in verification whose run-time
+   * depends on the message. It is safe to declassify here even with a secret
+   * message. */
```

Comment:
Please add a short comment _why_ it is safe?

Reply:
done.